pychoco/constraints/int_constraint_factory.py

```diff
@@ -1113,3 +1113,11 @@ def tree(self, successors: List[IntVar], nb_trees: IntVar, offset: int = 0):
         successors_handle = make_intvar_array(successors)
         constraint_handle = backend.tree(self.handle, successors_handle, nb_trees.handle, offset)
         return Constraint(constraint_handle, self)
+
+    def if_then(self, ifcons: Constraint, thencons: Constraint):
```

Comment:
Could you move this method in a new Python file called `reification_factory.py` and make the model extend in ? (in the same scheme as with int_constraint_factory, graph_constraint_factory, and set_constraint_factory). So that the code organization would be remain consistent with the Java version.